datasets/us/finra_enforcement_actions/us_finra_enforcement_actions.yaml

```diff
@@ -0,0 +1,26 @@
+title: FINRA Enforcement Actions
+entry_point: crawler.py
+prefix: us-finra-enfor
+coverage:
+  frequency: daily
+load_db_uri: ${OPENSANCTIONS_DATABASE_URI}
+summary: >
+  Firms and individuals that violated FINRA rules
+description: |
+  This dataset comprises firmas and individuals who FINRA have taken
+  disciplinary actions against for violations of its rules,
+  federal securities laws, rules, and regulations or
+  the rules of the Municipal Securities Rulemaking Board.
+  Financial Industry Regulatory Authority (FINRA)
+  is a private American corporation that acts as
+  a self-regulatory organization (SRO) that regulates
+  member brokerage firms and exchange markets. 
+publisher:
+  name: Financial Industry Regulatory Authority (FINRA)
+  url: https://www.finra.org/#/
+  official: false
+  country: "us"
+url: "https://www.finra.org/rules-guidance/oversight-enforcement/disciplinary-actions#:~:text=FINRA%20takes%20disciplinary%20actions%20against,the%20Municipal%20Securities%20Rulemaking%20Board."
+data:
+  url: https://www.finra.org/rules-guidance/oversight-enforcement/finra-disciplinary-actions?search=&firms=&individuals=&field_fda_case_id_txt=&field_core_official_dt%5Bmin%5D=&field_core_official_dt%5Bmax%5D=&field_fda_document_type_tax=All&token=f8W1uC3kpshSH5QzszZnszbMmQxZ77JDjlLYn16IuOw
```

Comment:
Does this cleaned up URL work just as well?

```suggestion
  url: https://www.finra.org/rules-guidance/oversight-enforcement/finra-disciplinary-actions
```

Reply:
There is just a slight difference, in the cleaned up URL, it has 18288, and in the original one it has 18286 because it filters specifically for firms and individuals. I found one of the two not related to firms and individuals and it looks like this:
<img width="1049" alt="Screenshot 2024-03-19 at 10 05 20" src="https://github.com/opensanctions/opensanctions/assets/37581842/79d64e04-c73d-4b5a-bd05-694ec6604d9c">
<img width="282" alt="Screenshot 2024-03-19 at 10 06 13" src="https://github.com/opensanctions/opensanctions/assets/37581842/32ceeed0-d20a-4e68-8878-3a6b747a3498">
<img width="329" alt="Screenshot 2024-03-19 at 10 06 18" src="https://github.com/opensanctions/opensanctions/assets/37581842/34fd36cf-a8cb-4a5a-a534-9b495d52cef8">

Because there is no icon, it is throwing an error in page 36. What do you want me to do? Use the cleaned up URL and ignore the lines without an icon? Or use the original url?

Reply:
Interesting! It doesn't work for me. I think let's use the cleaned up URL, and then skip if the firms-individuals cell doesn't contain a name? And if it has a name but no icon, you can leave it as LegalEntity